lib/tasks/poll.rake

```diff
@@ -38,19 +38,32 @@ namespace :poll do
         resp = get_post_req(encode_bbb_uri('getMeetings', server.url, server.secret))
         meetings = resp.xpath('/response/meetings/meeting')
 
+        total_attendees = 0
+        meetings_min_user_count = ENV['MEETINGS_MIN_USER_COUNT'] || 15
+        x_minutes_ago = (ENV['MEETING_JOIN_BUFFER_TIME'] || 15).minutes.ago
```

Comment:
Environment variables are string type, so you get some type errors here when the environment variables are set.

I recommend reading the environment variables in `config/application.rb` and handle parsing them to integers and default values there, like
```ruby
config.x.load_min_user_count = ENV.fetch('LOAD_MIN_USER_COUNT', 15).to_i
config.x.load_join_buffer_time = ENV.fetch('LOAD_JOIN_BUFFER_TIME', 15).to_i.minutes
```
then you can do
```ruby
x_minutes_ago = Rails.configuration.x.load_join_buffer_time.ago
```